portfolio.html

```diff
@@ -0,0 +1,219 @@
+        </div>
+        <div class="btn-languages ">
+            <button type="button" class="function">html</button>
+            <button type="button" class="function">bootstrap</button>
+            <button type="button"class="function">Ruby</button>
+        </div>
+    </div>
+    <div class="btn-allProjects">
+        <button type="button">See Project</button>
+    </div>
+</div>
+<div class="printing-data">
+    <div class="right-block">
+        <div class="printing-title">
+            <h1>Professional Art <br>Printing Data</h1>
+        </div>
+        <div class="post-content">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="btn-languages ">
+            <button type="button" class="function">html</button>
+            <button type="button" class="function">bootstrap</button>
+            <button type="button"class="function">Ruby</button>
+        </div>
+    </div>
+    <div class="btn-allProjects">
+        <button type="button">See Project</button>
+    </div>
+</div>
+<div class="printing-data">
+    <div class="right-block">
+        <div class="printing-title">
+            <h1>Professional Art <br>Printing Data</h1>
+        </div>
+        <div class="post-content">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="btn-languages ">
+            <button type="button" class="function">html</button>
+            <button type="button" class="function">bootstrap</button>
+            <button type="button"class="function">Ruby</button>
+        </div>
+    </div>
+    <div class="btn-allProjects">
+        <button type="button">See Project</button>
+    </div>
+</div>
+<div class="printing-data">
+    <div class="right-block">
+        <div class="printing-title">
+            <h1>Professional Art <br>Printing Data</h1>
+        </div>
+        <div class="post-content">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="btn-languages ">
+            <button type="button" class="function">html</button>
+            <button type="button" class="function">bootstrap</button>
+            <button type="button"class="function">Ruby</button>
+        </div>
+    </div>
+    <div class="btn-allProjects">
+        <button type="button">See Project</button>
+    </div>
+</div>
+<section class="aboutme">
+    <h2>About Me</h2>
+    <p class="software">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <hr>
```

Comment:
- [ ] Kindly consider taking a `Get my Resume` `button` or `a` tag and apply button styling on it.
- [ ] Also kindly consider laying out elements in this section using `display: flex` to follow best practices.